core-library/src/main/java/org/silverpeas/core/node/importexport/NodePositionType.java

```diff
@@ -26,15 +26,23 @@
  */
 package org.silverpeas.core.node.importexport;
 
-/**
- * Class used for Castor (un)marshalling
- * @author sdevolder
- */
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@XmlAccessorType(XmlAccessType.NONE)
 public class NodePositionType {
 
+  @XmlAttribute(name = "topicId")
   private int id = -1;
+  @XmlAttribute
   private String explicitPath;
 
+  public NodePositionType() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)